src/components/EmptyStateWrapper/EmptyStateWrapper.jsx

```diff
@@ -83,8 +83,8 @@ const DataTableWrapper = createClass({
 			isLoading,
 		} = this.props;
 
-		const emptyMessageBodyProp = _.get(getFirst(this.props, DataTableWrapper.EmptyMessageBody), 'props');
-		const emptyMessageTitleProp = _.get(getFirst(this.props, DataTableWrapper.EmptyMessageTitle), 'props', {children: 'You have no Line Items.'});
+		const emptyMessageBodyProp = _.get(getFirst(this.props, EmptyStateWrapper.EmptyMessageBody), 'props');
+		const emptyMessageTitleProp = _.get(getFirst(this.props, EmptyStateWrapper.EmptyMessageTitle), 'props', {children: 'You have no Line Items.'});
```

Comment:
I think we should change the default here from 'You have no Line Items.' to 'You have no data.'.